static_inventories/donbot_azure.yml

```diff
@@ -54,8 +54,14 @@ all:
           iptables_allow_ldap_inbound: "{{ iptables_allow_ssh_inbound }}"
           perun_facility_name: "umcg-gcc"
           perun_service_whitelist: "ldap_lsaai"
+administration:
+cluster:
+  children:
+    compute_node:
+    administration:
```

Comment:
Just curious: why was this added? I do not see any `compute_node` nor `administration` hosts in this inventory...

Reply:
I have deployed jumphost machine at some point, and then the playbook fails without this. This at least creates the structure, a pointer references if you wish. But since this is not breaking anything and is failing without, I decided to leave it in this PR.